### PR TITLE
config: enable all flow probes by default

### PR DESCRIPTION
**Pull request by masco-enovance**

### CODE REVIEW

**Comment by masco-enovance** [comment]
rerun tests

**Comment by lebauce** [comment]
@masco-enovance Can you please rebase on master ?

**Comment by lebauce** [comment]
ping @masco-enovance :-)

**Comment by masco-enovance** [comment]
rerun tests

File: etc/skydive.yml.default [hunk 1]
```diff
@@ -100,16 +100,6 @@ agent:
       # delay in seconds between two metric updates
       # metrics_update: 30
   flow:
-    # Probes used to capture traffic.
```
Comment (safchain) [inline etc/skydive.yml.default]: The flow probe section has to be removed from the test config file too in tests/tests.go

Comment (masco-enovance) [inline etc/skydive.yml.default]: done

Comment (safchain) [inline etc/skydive.yml.default]: arf sorry, forgot that but there is many other places, check in contrib folder and the devstack plugin

Comment (masco-enovance) [inline etc/skydive.yml.default]: i hope now removed all. thanks to the grep cmd.

**Pull request merged**